log/log.go

```diff
@@ -25,6 +25,16 @@ func (l Logger) With(keysvals ...interface{}) Logger {
 }
 
 /*
+WrapError can wrap error with kerrs to structured log
+*/
+func (l Logger) WrapError(err error) log.Logger {
```

Comment:
`WrapError` and `WithError` have the same signature and basically the same description. How would I choose which one to use?

Reply:
Any suggestions to improve the name? I think when you see `Wrap` you know it wraps.. then you know that `WithError` don't wrap.